client/components/edit/ChildList.tsx

```diff
@@ -1,19 +1,22 @@
 import React, { useEffect, useState } from "react";
-import PropTypes from "prop-types";
 import { useFetchContext } from "../../context/FetchContext";
 import { apiUrl } from "../../util/routes";
 import Link from "next/link";
 
-const ChildList = ({ pid = null }) => {
+interface ChildListProps {
+    pid: string;
+}
+
+const ChildList = ({ pid = "" }: ChildListProps) => {
     const {
         action: { fetchJSON },
     } = useFetchContext();
-    const [children, setChildren] = useState([]);
+    const [children, setChildren] = useState<any[]>([]);
```

Comment:
Can/should we be more specific than `any` here? What is the return type of fetchJSON? Can/should we use that?

Reply:
Made change.